From fc87b2d214cd7f6354f5899bdf704462d73bbb36 Mon Sep 17 00:00:00 2001
From: Dongwon Kim <dongwon.kim@intel.com>
Date: Mon, 17 Jun 2019 22:43:08 -0700
Subject: [PATCH 1441/1552] dma-buf/hyper_dmabuf: imported shouldn't be removed
 from the list if used

'imported' shouldn't be removed upon UNEXPORT_NOTIFY msg from exporter
if it's still being used by any importers because those importers may
need to retrieve it from the list.

Signed-off-by: Dongwon Kim <dongwon.kim@intel.com>
---
 drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_msg.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_msg.c b/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_msg.c
index ac0508b..d5a0afd 100644
--- a/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_msg.c
+++ b/drivers/dma-buf/hyper_dmabuf/hyper_dmabuf_msg.c
@@ -324,7 +324,6 @@ int hyper_dmabuf_msg_parse(int domid, struct hyper_dmabuf_req *req)
 				 * anymore.
 				 */
 				imported->valid = false;
-				hyper_dmabuf_remove_imported(hid);
 			} else {
 				/* No one is using buffer, remove it from
 				 * imported list
-- 
2.7.4

